src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -391,10 +396,13 @@ private function filterCatalogue(MessageCatalogue $catalogue, string $domain): M
         return $filteredCatalogue;
     }
 
-    private function extractMessages(string $locale, array $transPaths, string $prefix): MessageCatalogue
+    private function extractMessages(string $locale, array $transPaths, string $prefix, bool $blank): MessageCatalogue
     {
         $extractedCatalogue = new MessageCatalogue($locale);
         $this->extractor->setPrefix($prefix);
+        if (method_exists($this->extractor, 'blank')) {
+            $this->extractor->blank($blank);
+        }
```

Comment:
`elseif($blank) { throw usefull message; }` would help for sf/translation@5.4, or implementations that aren't upgraded yet

Reply:
Done 👍